app/controllers/spina/registers_controller.rb

```diff
@@ -54,36 +56,77 @@ def history
       @register = Spina::Register.find_by_slug!(params[:id])
       @register_data = @@registers_client.get_register(@register.name.parameterize, @register.register_phase)
 
-      fields = @register_data.get_field_definitions.map{ |field| field[:item]['field'] }
+      fields = @register_data.get_field_definitions.map { |field| field[:item]['field'] }
       all_records = @register_data.get_records_with_history
       entries_reversed = @register_data.get_entries.reverse
 
-      entries_mapped_with_items = entries_reversed.map { |entry|
+      entries_mapped_with_items = entries_reversed.map do |entry|
         records_for_key = all_records[entry[:key]]
-        current_record = records_for_key.detect{ |record| record[:entry_number] == entry[:entry_number] }
+        current_record = records_for_key.detect { |record| record[:entry_number] == entry[:entry_number] }
         previous_record_index = records_for_key.find_index(current_record) - 1
 
         if previous_record_index < 0
           changed_fields = fields
         else
           previous_record = records_for_key[previous_record_index]
-          changed_fields = fields.select { |f| current_record[:item][f] != previous_record[:item][f] }
+          changed_fields = fields.reject { |f| current_record[:item][f] == previous_record[:item][f] }
         end
 
         { current_record: current_record, previous_record: previous_record, updated_fields: changed_fields, key: entry[:key] }
-      }
+      end
 
-      @entries_with_items = paginate(entries_mapped_with_items)
+      if params[:search_input].present?
```

Comment:
I still think we should make this `q`. At least then we are consistent. Maybe if we come up with a better name then we can update it everywhere.